### PR TITLE
Manage Teku's JVM memory usage.

### PR DESCRIPTION
**Pull request by bgravenorst**
Signed-off-by: Byron Gravenorst <byron.gravenorst@consensys.net>

## Pull Request Description
Add content to manage Teku's JVM memory usage.

### CODE REVIEW

File: docs/HowTo/Get-Started/Manage-Memory.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+=== "Command line"
+
+    ```bash
+    JAVA_OPTS=-Xmx2g ./teku [options]
+    ```
+
+!!! note
+
+    The node uses of more RAM to perform better if it’s available, especially during periods
+    of non-finalization.
```
Comment (ajsutton) [inline docs/HowTo/Get-Started/Manage-Memory.md:30]:
```suggestion
    The node uses more RAM to perform better if it’s available, especially during periods
    of non-finalization.
```

File: docs/HowTo/Get-Started/Manage-Memory.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
+---
+title: Teku memory management
+---
+
+# Managing memory
+
+Manage Teku's Java Virtual Machine (JVM) memory requirements by setting a maximum heap size
+using the `JAVA_OPTS` environment variable.
+
+We recommend setting the maximum heap size to 2GB if you have less than 8GB of RAM. This results in
```
Comment (ajsutton) [inline docs/HowTo/Get-Started/Manage-Memory.md:10]: Might be worth noting that setting the max heap is also useful if you have a lot of RAM and want Teku to use less of it to leave more room for other things.

**Review by ajsutton — APPROVED** [review]
LGTM.

File: docs/HowTo/Get-Started/Manage-Memory.md (hunk 2 above)
Comment (bgravenorst) [reply docs/HowTo/Get-Started/Manage-Memory.md:10]: Done

**Pull request merged**